public/app/features/connections/tabs/ConnectData/CardGrid/CardGrid.tsx

```diff
@@ -64,7 +58,7 @@ export const CardGrid = ({ items, onClickItem }: CardGridProps) => {
   const styles = useStyles2(getStyles);
 
   return (
-    <ul className={styles.sourcesList}>
+    <Grid display="grid" templateColumns="repeat(auto-fill, minmax(380px, 1fr))" gap={1.5}>
```

Comment:
Comparing the view in ops and locally it seems that there is a lot of unnecessary empty space now because the cards are arranged in only 2 columns until the breakpoint of approx. 1500px (before there were three, breakpoint from 2 to 3 columns 1250px). Is this intentional?

View on localhost:
<img width="1170" alt="Screenshot 2023-09-19 at 17 40 57" src="https://github.com/grafana/grafana/assets/48948963/6f8cd693-4010-4422-9a6f-629cf5db9558">

View in ops:
<img width="1175" alt="Screenshot 2023-09-19 at 17 41 46" src="https://github.com/grafana/grafana/assets/48948963/bb197c91-3fd6-406e-84cc-17a3ee855517">

Reply:
This seems to be a general issue. I talked to @teddyba as well as @besartberisha and I created an issue for the responsible team: https://github.com/grafana/cloud-onboarding/issues/5013